tests/checkpoint/test_checkpoint_result_format.py

```diff
@@ -2851,76 +2766,6 @@ def test_pandas_result_format_in_checkpoint_named_index_two_index_column_not_set
     assert first_result_partial_list == [("three"), ("four"), ("five")]
 
 
```

Comment:
@Kilo59 This one was a duplicate for some reason.. totally good to delete the second one.

Reply:
I'm going to save these all for a subsequent PR.